src/plugins/waitlist.js

```diff
@@ -31,14 +31,33 @@ export default function waitlistPlugin () {
       }
     }
 
-    function onModAddDj () {
-      // TODO
+    function onModAddDj (ref) {
+      mp.emit('modAddDj', {
+        mod: mp.user(ref.mi) || mp.wrapUser({ id: ref.mi }),
+        user: mp.wrapUser({ username: ref.t }),
```

Comment:
I suppose the event from plug.dj doesn't include a user ID? :( a user object with only the `username` isn't that useful since none of the methods would work anyway… it may be better to just expose a `username` property on the event instead.